js/src/scrollspy.js

```diff
@@ -269,7 +269,7 @@ const ScrollSpy = (($) => {
     static _jQueryInterface(config) {
       return this.each(function () {
         let data    = $(this).data(DATA_KEY)
-        let _config = typeof config === 'object' && config || null
```

Comment:
Actually looks like I was wrong and the new version will also return `null`. But yeah I agree that overall this PR doesn't add much value if any.